tests/monitoring_integration_test.py

```diff
@@ -95,16 +95,23 @@ def testUptimeCheckAlertingPolicyNotificationChannel(self):
 		self.assertEqual(len(channel_list), 1)
 
 class TestMonitoringDashboard(unittest.TestCase):
-	def testUserExpDashboard(self):
-		""" Test that the User Experience Dashboard gets created. """
+	def checkForDashboard(self, dashboard_display_name):
 		client = v1.DashboardsServiceClient()
 		dashboards = client.list_dashboards(project_name)
-		found_userexp_dashboard = False
 		for dashboard in dashboards:
-			if dashboard.display_name == 'User Experience Dashboard':
-				found_userexp_dashboard = True
-				
-		self.assertTrue(found_userexp_dashboard)
+			if dashboard.display_name == dashboard_display_name:
+				return True
+		return False
+
+	def testUserExpDashboard(self):
+		""" Test that the User Experience Dashboard gets created. """
+		found_dash = self.checkForDashboard('User Experience Dashboard')
+		self.assertTrue(found_dash)
+
+	def testFrontendServiceDashboard(self):
+		""" Test that the Frontend Service Dashboard gets created. """
+		found_dash = self.checkForDashboard('Frontend Service Dashboard')
+		self.assertTrue(found_dash)
```

Comment:
Are these tests added to run as part of each CI run?

Reply:
They're not - not sure if you saw the conversation with Dan in the chat room but basically we think that setting up the testing environment will take significant changes that might be better to be added in the next sprint/release. The tests are still run manually but they are passing.